### PR TITLE
Stores

### LINKED ISSUE #29: Create a page with all stores
Add also a pagination
Follow:https://github.com/Abbe-and-Gabbe/jkpgCity/blob/main/JKPGCITY.pdf

### PR DESCRIPTION
**Pull request by GabrielAdward**

### CODE REVIEW

File: frontend/src/Components/Footer.svelte [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+<footer class="bottom-0 left-0 z-20 w-full p-4 bg-gray-50 dark:bg-gray-900 border-t border-gray-200 shadow md:flex md:items-center md:justify-between md:p-6 dark:border-gray-600">
     <span class="text-sm text-gray-500 sm:text-center dark:text-gray-400">© 2024 <a href="/" class="hover:underline">JKPGCITY</a>. All Rights Reserved.
     </span>
     <ul class="flex flex-wrap items-center mt-3 text-sm font-medium text-gray-500 dark:text-gray-400 sm:mt-0">
         <li>
             <a href="#" class="hover:underline me-4 md:me-6">Home</a>
         </li>
         <li>
-            <a href="#" class="hover:underline me-4 md:me-6">Venues</a>
+            <a href="/stores" class="hover:underline me-4 md:me-6">Stores</a>
```
Comment (albin02forsberg) [inline frontend/src/Components/Footer.svelte]: Should be using `<Link>` tag?

**Review by albin02forsberg — CHANGES_REQUESTED** [review]
Small stuff ;)

File: frontend/src/Components/Footer.svelte (hunk 1 above)
Comment (GabrielAdward) [reply frontend/src/Components/Footer.svelte]: Don't think so i googled and looked through our old project, it did not contain any link

**Comment by GabrielAdward** [comment]
Ready for review!

**Review by saleelala — APPROVED** [review]
LGTM

**Review by albin02forsberg — APPROVED** [review]

**Pull request merged**